.github/workflows/test-verify-html.yaml

```diff
@@ -0,0 +1,40 @@
+name: Test verify-html dispatch
+# Workflow Update OCM CLI Docs now has two triggers:
+# 1. pull_request: branches: main
+# 2. workflow_dispatch: pr_number
+# since workflows of kind pull_request are not triggered by the GITHUB_TOKEN
```

Comment:
Is this something you actually verified or just did on a hunch?

Reply:
this action is to test what I want to merge before the merge. Turns out to be not so easy, but I guess now I have it. Testing actions in branches sucks, especially when it is about repository_dispatch :-) I will remove the workflow from the PR once the test was successful.